media-libs/fcft/fcft-3.0.0.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit meson python-any-r1
+
+DESCRIPTION="Simple library for font loading and glyph rasterization"
+HOMEPAGE="https://codeberg.org/dnkl/fcft"
+SRC_URI="https://codeberg.org/dnkl/fcft/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+harfbuzz +libutf8proc examples test"
+REQUIRED_USE="
+	libutf8proc? ( harfbuzz )
+	examples? ( libutf8proc )
+"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	media-libs/fontconfig
+	media-libs/freetype
+	x11-libs/pixman
+	harfbuzz? (
+		media-libs/harfbuzz:=
+	)
+	libutf8proc? (
+		dev-libs/libutf8proc:=
+	)
```

Comment:
I tried to build with `USE="harfbuzz -libutf8proc"` and
```
 * VDB: detected possibly incorrect RDEPEND (media-libs/fcft-3.0.0)
 *  > dev-libs/libutf8proc:=
```
So harfbuzz always need it I guess? At this point makes me feel like the two USE should be combined.